docs/website/binding_objc_libs.md

```diff
@@ -840,6 +840,51 @@ An alternative is to mark the original, `NSString`-based, definition as
 the wrap'ed version will still work and call the overriden method.
 
 
+## Binding NSValue NSNumber and NSString to a better type
+
+The [[BindAs]](/guides/ios/advanced_topics/binding_objective-c/binding_types_reference_guide#BindAsAttribute) attribute allows binding `NSNumber` `NSValue` and `NSString`(enums) into more accurate C# types this allows an easy way to create better .NET API.
+
+It can be used in methods, parameters and properties. The only restriction is that your member **mustn't** be inside a `[Protocol]` or `[Model]` interface.
```

Comment:
**must not** be part of an interface decorated with a `[Protocol]` or a` [Model]` attribute.